### PR TITLE
all done

### PR DESCRIPTION
**Pull request by danieltichiyama**
finished assignment. ready for review.

### CODE REVIEW

File: secret_number.js [hunk 1]
```diff
@@ -1,4 +1,18 @@
 'use strict';
 module.exports = function() {
-  
+
+let num;
+
+// = Math.floor(Math.random()*1000001);
```
Comment (taesup) [inline secret_number.js]: Remove commented code.

File: secret_number.js [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+
+(function randomizer (){
```
Comment (taesup) [inline secret_number.js]: This doesn't need to be in an IIFE

File: secret_number.js [hunk 3]
```diff
@@ -1,4 +1,18 @@
...
+    num = Math.floor(Math.random()*1000001);
+})();
+    
+return function (){
+    // let num = Math.floor(Math.random()*1000001);
```
Comment (taesup) [inline secret_number.js]: Remove commented code.

File: secret_number.js [hunk 4]
```diff
@@ -1,4 +1,18 @@
...
+    console.log(num);
```
Comment (taesup) [inline secret_number.js]: Remove console.log

**Review by taesup — CHANGES_REQUESTED** [review]

File: secret_number.js (hunk 2 above)
Comment (danieltichiyama) [reply secret_number.js]: without it being an IIFE, it never gets called and the value of num is never set to a random number...

Comment (danieltichiyama) [reply secret_number.js]: okay, change made.

**Review by taesup — APPROVED** [review]

**Pull request merged**